apps/demo/app/routes/index.tsx

```diff
@@ -1,34 +1,39 @@
-// ignore all type errors
-// @ts-nocheck - This needs to be rebuilt
-
-import React from 'react';
+import React, { ComponentType, SVGProps } from 'react';
+import Github from '../../../../library/foundations/generated/icons/github';
+import Figma from '../../../../library/foundations/generated/icons/figma';
+import EventNote from '../../../../library/foundations/generated/icons/event-note';
+// import Github from '@rangle/radius-foundations/generated/icons/github.tsx';
+// import Figma from '@rangle/radius-foundations/generated/icons/figma';
+// import EventNote from '@rangle/radius-foundations/generated/icons/event-note';
```

Comment:
To import them correctly we'll need to change to the `@rangle/radius-foundations` which will take a while.